jobs/config.json

```diff
@@ -21,10 +21,11 @@
   },
   "ci-kops-build": {
     "args": [
-      "--script=make gcs-publish-ci",
-      "--kops=gs://kops-ci/bin"
+      "make",
+      "gcs-publish-ci",
+      "GCS_LOCATION=gs://kops-ci/bin"
```

Comment:
The above syntax is fine for `make` variables, e.g. https://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make